.github/workflows/gh-pages.yml

```
@@ -0,0 +1,28 @@
+name: gh-pages
+on:
+  push:
+    branches: [v2]
+
+env:
+  NODEJS_VERSION: "16"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODEJS_VERSION }}
+          cache: npm
+      - run: npm install
+      - name: Bundle specification
+        run: npm run build
```

Тогда наверно?
```suggestion
        run: npm run build -- ./dist
```
Потому что https://github.com/valitydev/swag-payments/blob/0089567101a77cab1c1bf324a34d314788449e33/scripts/build.js#L5

---

Не знаю, где лучше подтюнить)